app/assets/javascripts/components/students/shared/StudentList/StudentDrawer/TrainingStatus/TrainingModuleRows.jsx

```diff
@@ -13,7 +13,9 @@ export const TrainingModuleRows = ({ trainingModules }) => {
     if (trainingModule.completion_date) {
       moduleStatus = (
         <span className="completed">
-          Completed at {moment(trainingModule.completion_date).format('YYYY-MM-DD   h:mm A')}
+          Completed at {moment(trainingModule.completion_date).format('YYYY-MM-DD   h:mm A')}.
+          <br/>
+          Time taken to complete {trainingModule.completion_time}.
```

Comment:
This should be done with `i18n`. (So should the "Completed at" part that is already present.)

Maybe "Completion time:" instead of "Time taken to complete", just to be a like more concise.

Reply:
Done! Please have a look over it.